honcho-cli/scripts/generate_cli_docs.py

```
@@ -0,0 +1,128 @@
+            lines.append(f"- {_format_param(p)}")
+        lines.append("")
+
+    if opts:
+        lines.append("**Options**\n")
+        for p in opts:
+            lines.append(f"- {_format_param(p)}")
+        lines.append("")
+
+    if isinstance(cmd, click.Group):
+        for sub_name in sorted(cmd.commands):
+            lines.extend(_render(cmd.commands[sub_name], path + [sub_name]))
```

Ruff (0.15.10)

[warning] Consider `[*path, sub_name]` instead of concatenation

Replace with `[*path, sub_name]`

(RUF005)